pkg/cmd/plan.go

```diff
@@ -23,56 +23,57 @@ import (
 
 var PlanCommand = cli.Command{
 	Name:  "plan",
-	Usage: "plan management",
+	Usage: "manage the plans known to the client",
 	Subcommands: cli.Commands{
 		&cli.Command{
 			Name:  "create",
-			Usage: "create a plan named `PLAN_NAME`",
+			Usage: "creates a new test plan, using a template from github.com/testground/plan-templates",
 			Flags: []cli.Flag{
 				&cli.StringFlag{
 					Name:     "remote",
-					Usage:    "origin for your repository i.e. git@github.com:your/repo",
+					Usage:    "`URL` of the repo where this plan will be hosted i.e. git@github.com:your/repo",
```

Comment:
I intentionally didn't use URL here since git origins are not necessarily URLs. They might be SSH, or paths to local bare repos, etc.

Reply:
Funny thing is that git docs calls them URLs!

![image](https://user-images.githubusercontent.com/1101242/81099364-b6ee5280-8f02-11ea-80d9-e986709b70fc.png)